x-pack/plugins/task_manager/server/task_running/errors.ts

```diff
@@ -13,10 +13,17 @@ const CODE_SKIP = 'TaskManager/skip';
 
 const code = Symbol('TaskManagerErrorCode');
 const retry = Symbol('TaskManagerErrorRetry');
+const source = Symbol('TaskManagerErrorSource');
 
-export interface DecoratedError extends Error {
```

Comment:
To cut down on the changes noise, I think we could probably leave this named `DecoratedError`.  I'm not sure there's a whole lot of value in the rename.